### PR TITLE
v1.16 changelog

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[KubOS 1.16](https://trello.com/c/9Pfnwzhk/1025-kubos-116)

File: changelog.rst [hunk 1]
```diff
@@ -1,6 +1,18 @@
 Kubos Linux Changelog
 =====================
 
+v1.16 - July 10 2019
+--------------------
+
+- `Set up release image auto-generation for the iOBC <https://github.com/kubos/kubos-linux-build/blob/master/board/kubos/at91sam9g20isis/post-image.sh>`__
```
Comment (jacoffey3) [inline changelog.rst]:
```suggestion
- Massively overhauled and expanded the documentation for KubOS
- `Set up release image auto-generation for the iOBC <https://github.com/kubos/kubos-linux-build/blob/master/board/kubos/at91sam9g20isis/post-image.sh>`__
```

Comment (CatGarab) [reply changelog.rst]: Ehhhh, yes, but that's not really related to this repo

Comment (jacoffey3) [reply changelog.rst]: Heh, I misread lol

**Pull request merged**